README.md

```diff
@@ -461,6 +460,18 @@ The following table lists the _latest_ eDSF Kit releases, their release date and
       <br/>3. Improvements and bug fixes.
     </td>
   </tr>
+  <tr>
+    <td>4 Apr 2024
+    </td>
+    <td>1.7.12
+    </td>
+    <td>
+      1. Added support for DRA version 4.15.0.11 (following 4.15.0.10 rollback).
+      <br/>2. Change default size of rds-mysql-db.
```

Comment:
Changed the default version and size of the MySQL RDS which is onbaorded to the DSF Hub in POC examples.